From aa05cc4e9fc4c83a04f2a49932ebad3ed6dfad3c Mon Sep 17 00:00:00 2001
From: "duncan.macleod" <duncan.macleod@ligo.org>
Date: Wed, 2 Aug 2023 17:45:35 +0100
Subject: [PATCH 1/3] setup.py: remove lalsuite install_requires

the conda package can depend upon the LALSuite subpackages individually
---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index 6a3c63faf..5961e4f4c 100755
--- a/setup.py
+++ b/setup.py
@@ -48,7 +48,6 @@ install_requires = setup_requires + [
     'pegasus-wms.api >= 5.0.8',
     'python-ligo-lw >= 1.7.0',
     'igwn-segments',
-    'lalsuite!=7.2',
     'lscsoft-glue>=1.59.3',
     'pykerr',
 ]
-- 
2.39.5

